### PR TITLE
fix include paths in all the -zkp modules

### PR DESCRIPTION
**Pull request by apoelstra**
This is causing out-of-tree build failures in Elements.

### CODE REVIEW

**Comment by apoelstra** [comment]
Specifically `make check` fails when building the exhaustive check binary.

**Comment by psgreco** [comment]
ack 0f036596659a582c5ec4cd0da9c2ec2ba316098c `make check` works in out-of-tree elements with this fix

File: src/modules/ecdsa_adaptor/main_impl.h [hunk 1]
```diff
@@ -7,8 +7,8 @@
 #ifndef SECP256K1_MODULE_ECDSA_ADAPTOR_MAIN_H
 #define SECP256K1_MODULE_ECDSA_ADAPTOR_MAIN_H
 
-#include "include/secp256k1_ecdsa_adaptor.h"
-#include "modules/ecdsa_adaptor/dleq_impl.h"
+#include "../../../include/secp256k1_ecdsa_adaptor.h"
+#include "./dleq_impl.h"
```
Comment (real-or-random) [inline src/modules/ecdsa_adaptor/main_impl.h:11]: nit:

```suggestion
#include "dleq_impl.h"
```
(and many other lines)

For consistency with upstream secp256k1, which doesn't have the "./" but I'm happy to ACK either way.

**Review by real-or-random — APPROVED** [review]
ACK https://github.com/ElementsProject/secp256k1-zkp/pull/197/commits/0f036596659a582c5ec4cd0da9c2ec2ba316098c

File: src/modules/ecdsa_adaptor/main_impl.h (hunk 1 above)
Comment (apoelstra) [reply src/modules/ecdsa_adaptor/main_impl.h:11]: Ok, cool, I'll remove the `./`s (in my local testing they don't seem to matter)

**Review by real-or-random — APPROVED** [review]
utACK 347f96d94a6c2840e045510672549897be743101

**Comment by apoelstra** [comment]
@psgreco sorry could you test one more time just to confirm this fixes Elemenst?

**Comment by psgreco** [comment]
ack 347f96d94a6c2840e045510672549897be743101 new patch works too

**Pull request merged**